pyramid_zipkin/tween.py

```diff
@@ -73,8 +75,14 @@ def _get_settings_from_request(request):
 
     if 'zipkin.transport_handler' in settings:
         transport_handler = settings['zipkin.transport_handler']
-        stream_name = settings.get('zipkin.stream_name', 'zipkin')
-        transport_handler = functools.partial(transport_handler, stream_name)
+        if not isinstance(transport_handler, BaseTransportHandler):
+            warnings.warn(
```

Comment:
I'm not a big fan of deprecation warnings. I'd either keep supporting both (why not give people the option?) or cut a release which removes that compatibility. In my experience these kinds of warnings tend to get ignored :(

Reply:
>  I'd either keep supporting both

Mostly because it's confusing. pyramid_zipkin's transport handler function receives 2 arguments, while py_zipkin explicitly says that it receives only one. That means you'll need to have 2 different transport implementations, one for py_zipkin and one for pyramid_zipkin.

Reply:
I was the one that suggested the deprecation warning and I still think that it is the right thing to do, even if we aren't at a 1.0 version and even if people ignore these warnings. The ideal situation for me would be to have an official API deprecation policy, but in the meantime I think we can be Good Citizens and warn people of an impending deprecation. Even if they ignore it, if they bump the version and things break, they can look back at previous builds and see the deprecation warning to better understand why things broke.

I dunno, maybe this is just the starry-eyed idealist in me talking...